server/terraform/developer-guide.md

```diff
@@ -0,0 +1,29 @@
+## How to create and setup GCP projects using terraform
+
+### Prerequisites
+
+- [Install Terraform](https://learn.hashicorp.com/tutorials/terraform/install-cli?in=terraform/gcp-get-started)
```

Comment:
Update `server/README.md` to add the terraform install step. Devs will need this for server work to validate changes even if they don't edit the Terraform files.